src/main/java/com/iemr/common/controller/users/IEMRAdminController.java

```diff
@@ -275,7 +285,8 @@ public String superUserAuthenticate(
 			if (!m_User.getUserName().equalsIgnoreCase("SuperAdmin")) {
 				throw new IEMRException("Please log with admin credentials");
 			}
-			User mUser = iemrAdminUserServiceImpl.superUserAuthenticate(m_User.getUserName(), m_User.getPassword());
+			String decryptPassword = aesUtil.decrypt("Piramal12Piramal", m_User.getPassword());
```

Comment:
@ravishanigarapu The username seems to be hardcoded here. Please check and correct.